packages/teleport/src/Discover/SelectResource/ServerResource.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { Link, Text } from 'design';
+
+import { ActionButtons, TextBox } from 'teleport/Discover/Shared';
+
+import { PermissionsErrorMessage } from './PermissionsErrorMessage';
+
+interface ServerResourceProps {
+  disabled: boolean;
+  onProceed: () => void;
+}
+
+export function ServerResource(props: ServerResourceProps) {
+  let content = <TeleportVersions />;
+  if (props.disabled) {
+    content = (
+      <PermissionsErrorMessage
+        action="add new Servers"
+        productName="Server Access"
+      />
+    );
+  }
+
+  return (
+    <>
+      {content}
+
```

Comment:
nit: this and other files, remove new line

Reply:
Personally I like the extra line here as it's a separator between dynamic content and JSX - is there a heavy preference here?

Reply:
nope, i always found "nits" to be more "optional", or maybe that's just me